src/sql/workbench/contrib/editData/browser/editDataResultsEditor.ts

```diff
@@ -102,24 +114,70 @@ export class EditDataResultsEditor extends BaseEditor {
 			throw new Error('DataService not found for URI: ' + uri);
 		}
 
+		//Stuff afterwards can be ignored
```

Comment:
not sure the comment, but assume it's related to WIP?

Reply:
That's correct.